frontend/src/metabase-lib/parameters/utils/parameter-fields.ts

```diff
@@ -4,18 +4,18 @@ import type {
   UiParameter,
 } from "metabase-lib/parameters/types";
 
-export const hasFields = (
+export const isFieldFilterUiParameter = (
   parameter: UiParameter,
 ): parameter is FieldFilterUiParameter => {
-  return (
-    "fields" in parameter &&
-    Array.isArray(parameter.fields) &&
-    parameter.fields.length > 0
-  );
+  return "fields" in parameter && Array.isArray(parameter.fields);
+};
+
+export const hasFields = (parameter: UiParameter): boolean => {
```

Comment:
```suggestion
export const hasFields = (parameter: UiParameter) => {
```

it's clear it's boolean